orchestra/tasks.py

```diff
@@ -0,0 +1,369 @@
+from datetime import datetime
+from logging import getLogger
+from pathlib import Path
+
+import dill
+import luigi
+import yaml
+from luigi.util import inherits, requires
+
+from epimodel import Level, RegionDataset, algorithms, imports, read_csv_smart, utils
+from epimodel.exports.epidemics_org import process_export, upload_export
+from epimodel.gleam import Batch, GleamDefinition, batch as batch_module
+
+log = getLogger(__name__)
+
+
+def default_from_config(task_name: str, param_name: str) -> dict:
+    return dict(section=task_name, name=param_name)
+
+
+class RegionsFile(luigi.ExternalTask):
```

Comment:
`luigi.ExternalTasks` are a luigi way of saying "there was something happening not covered in this codebase, e.g. by human, which produces my `output`"